### PR TITLE
Use symbolic operator names (moveto, lineto) in contour_manual example.

### PR DESCRIPTION
**Pull request by anntzer**
It's clearer to directly use the names (or their one-letter abbreviations M/L, which are e.g. part of the svg standard) that using numeric values and then explaining the numeric values.

## PR Checklist

**Documentation and Tests**
- [ ] Has pytest style unit tests (and `pytest` passes)
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] New plotting related features are documented with examples.

**Release Notes**
- [ ] New features are marked with a `.. versionadded::` directive in the docstring and documented in `doc/users/next_whats_new/`
- [ ] API changes are marked with a `.. versionchanged::` directive in the docstring and documented in `doc/api/next_api_changes/`
- [ ] Release notes conform with instructions in  `next_whats_new/README.rst` or `next_api_changes/README.rst`

### CODE REVIEW

File: galleries/examples/misc/contour_manual.py [hunk 1]
```diff
@@ -43,11 +44,12 @@
...
 # vertices along with a list of vertex kinds (code types) as described in the
 # Path class.  This is particularly useful for polygons with holes.
-# Here a code type of 1 is a MOVETO, and 2 is a LINETO.
 
 fig, ax = plt.subplots()
 filled01 = [[[0, 0], [3, 0], [3, 3], [0, 3], [1, 1], [1, 2], [2, 2], [2, 1]]]
-kinds01 = [[1, 2, 2, 2, 1, 2, 2, 2]]
+M = Path.MOVETO
+L = Path.LINETO
+kinds01 = [[M, L, L, L, M, L, L, L]]
```
Comment (timhoffm) [inline galleries/examples/misc/contour_manual.py:52]: While at it, I would go a bit more verbose:

```suggestion
MOVE = Path.MOVETO
LINE = Path.LINETO
kinds01 = [[MOVE, LINE, LINE, LINE, MOVE, LINE, LINE, LINE]]
```

or even

```
steps = [
    (Path.MOVETO, [0, 0]),
    (Path.LINETO, [3, 0]),
    (Path.LINETO, [3, 3]),
    (Path.LINETO, [0, 3]),
    (Path.MOVETO, [1, 1]),
    (Path.LINETO, [1, 2]),
    (Path.LINETO, [2, 2]),
    (Path.LINETO, [3, 1]),
]

fills, kinds = zip(*steps)
```

But the proposal is also ok if you don't want to do more.

**Review by timhoffm — APPROVED** [review]
Take or leave my remark.

File: galleries/examples/misc/contour_manual.py (hunk 1 above)
Comment (anntzer) [reply galleries/examples/misc/contour_manual.py:52]: I don't think long names really help readability here; the point is really to see that there's 1 M followed by 3 Ls, twice.

Comment (oscargus) [reply galleries/examples/misc/contour_manual.py:52]: I think that some way to "visually" connect the code and the points would be nice, but I am not really sure the zip-thing is worth it. So may just keep it as is.

File: galleries/examples/misc/contour_manual.py [hunk 2]
```diff
@@ -5,10 +5,11 @@
 
 Example of displaying your own contour lines and polygons using ContourSet.
 """
-import matplotlib.pyplot as plt
 
 import matplotlib.cm as cm
 from matplotlib.contour import ContourSet
+import matplotlib.pyplot as plt
+from matplotlib.path import Path
```
[the remark this answers is not in the document]
Comment (anntzer) [reply galleries/examples/misc/contour_manual.py:13]: fixed

**Comment by timhoffm** [comment]
Let’s take it as is.

**Pull request merged**